<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>611</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar se√±al</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mostrar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>470</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar segemento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="minimo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="maximo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>430</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Minimo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Maximo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_regresar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>271</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
